packages/alfa-rules/test/sia-r11/rule.spec.tsx

```diff
@@ -0,0 +1,47 @@
+import { test } from "@siteimprove/alfa-test";
+import { Document } from "@siteimprove/alfa-dom";
+
+import R11, { Outcomes } from "../../src/sia-r11/rule";
+
+import { evaluate } from "../common/evaluate";
+import { passed, failed, inapplicable } from "../common/outcome";
+
+test(`evaluate() passes a link with a name given by content`, async (t) => {
+  const target = <a href="#">Foo</a>;
+
+  const document = Document.of([target]);
```

Comment:
This should use the `h.document()` API instead. We should actually go through all existing tests at some point and change this. The HyperScript-like `h` API is designed for convenience, unlike the DOM constructor functions, and is therefore favoured for tests.

Reply:
Feels like this updated all the tests 😅 https://github.com/Siteimprove/alfa/pull/771/commits/ce61ed613795fbe131cffa8a462fffd141efa171